packages/python/src/create-module.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  type Binder,
+  getSymbolCreatorSymbol,
+  type Refkey,
+  refkey,
+  SymbolCreator,
+} from "@alloy-js/core";
+import { PythonModuleScope, PythonOutputSymbol } from "./symbols/index.js";
+
+export interface ModuleDescriptor {
+  [path: string]: ModuleSymbolsDescriptor;
+}
+
+export type NamedModuleDescriptor = string;
+
+export interface ModuleSymbolsDescriptor {
+  named?: NamedModuleDescriptor[];
+}
+
+export interface CreateModuleProps<T extends ModuleDescriptor> {
+  name: string;
+  version: string;
+  descriptor: T;
+}
+
+export type ModuleExports<
```

Comment:
I don't know if this should be called ModuleExports. In python everything is exported, so maybe this should be called NamedModules or something